### PR TITLE
docs: README

### PR DESCRIPTION
**Pull request by hannessolo**
Replaces https://github.com/adobe-rnd/franklin/pull/71, https://github.com/adobe-rnd/franklin/pull/32
Closes https://github.com/adobe-rnd/franklin/issues/56, https://github.com/adobe-rnd/franklin/issues/70

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Franklin
+
+## What is Franklin?
+
+Franklin is the next-generation rendering pipeline for AEM Sites. We aim to provide a great modern developer experience as well as 100% lighthouse scores out of the box.
```
Comment (icaraps) [inline readme.md:5]: Maybe dont mention AEM Site for the moment

**Comment by icaraps** [comment]
You forgot to mention dynamic routes you worked one ? 😄

**Comment by hannessolo** [comment]
@icaraps updated

**Comment by icaraps** [comment]
Oh I just noticed we didn't say anything about `app.config.yaml` ? I think we should mention it and say that tenant, project, cache, actions and which pages to deploy can be specified in there WDYT ?

**Comment by hannessolo** [comment]
> Oh I just noticed we didn't say anything about `app.config.yaml` ? I think we should mention it and say that tenant, project, cache, actions and which pages to deploy can be specified in there WDYT ?

Is pages actually the pages that are deployed, or is it just the ones that lighthouse scoring is run for?

But yes, I'll add it. So many features, I forget them all 😃

**Comment by icaraps** [comment]
Maybe you can also add in the components section that we have default components based on https://github.com/adobe/aem-site-template-standard that work with default [models](https://core.project-franklin-api.workers.dev/model?tenant=demo-site&path=/content/demo-site/en/home) published under the `demo-site` tenant for pages like `/content/demo-site/en/home`

**Pull request merged**